source/opencascade/utilities.cc

```diff
@@ -613,7 +613,8 @@ namespace OpenCASCADE
     Assert(props.IsNormalDefined(), ExcMessage("Normal is not well defined!"));
     gp_Dir Normal = props.Normal();
     Assert(props.IsCurvatureDefined(), ExcMessage("Curvature is not well defined!"));
-    Standard_Real Mean_Curvature = props.MeanCurvature();
+    Standard_Real Min_Curvature = props.MinCurvature();
+    Standard_Real Max_Curvature = props.MaxCurvature();
     Point<3> normal = Point<3>(Normal.X(),Normal.Y(),Normal.Z());
```

Comment:
This would be the place to represent the normal as a tensor rather than a point. (Because a point can be converted to a tensor, no need to adjust the right hand side - only the type on the left. )